### ISSUE TITLE
Cache without playing

### ISSUE THREAD

**Issue by NikitaVashchenko** [VeinGuo/VGPlayer#48]
Hi, Please explain to me the way to download video directly to the cache without playing with VGPlayer. Problem is you don't use comments so it's pretty hard to understand your code. Could you add any explaining about how the cache is managed under the hood and how to manage cached videos directly not only "cleanAllCache()".

**Comment by userSV** [comment]
@NikitaVashchenko Did you manage to find the solution for this?

**Comment by NikitaVashchenko** [comment]
> @NikitaVashchenko Did you manage to find the solution for this?

No, I decided to make my own media cache manager for my purposes and did it successfully
